packages/theme/modules/wishlist/getters/wishlistGetters.ts

```diff
@@ -49,6 +49,7 @@ export const getTotals = (wishlist: Wishlist[] | Wishlist): Totals => {
   }), ({ total: 0, subtotal: 0 }));
 };
 
+// TO-DO: remove
```

Comment:
If you do it like this your IDE will be able to trace it (at least webstorm can)
```suggestion
// TODO: remove
```

Also, please add more descriptive reason.